### ISSUE TITLE
Slim support (ruby slim)

### ISSUE THREAD

**Issue by riley-usagi** [Glavin001/atom-beautify#634]
This would be awesome.

**Comment by phoenixweiss** [comment]
I think the same way!

**Comment by outwitevil** [comment]
Me too

**Comment by prettydiff** [comment]
The language appears to be: http://slim-lang.com/ which looks similar to Jade and would require a similar beautifier.

**Comment by rtack** [comment]
Were the changes in https://github.com/Glavin001/atom-beautify/commit/d48768e5034e59f2c618c7d56c05693879441124 all that is required to add support for Jade? Or is the comment in #199 "I haven't had a chance to finish adding Jade support. However, I have moved this back to v0.29.0 milestone, so I want it to be in the next release." indicating the support is not fully implemented yet?

**Comment by prettydiff** [comment]
@Glavin001 could answer the prior questions better.

**Comment by Glavin001** [comment]
@rtack : That commit does add support for Jade. Before I made that comment I had not implemented Jade support, however after seeing how many users requested it and also that Pug-beautify package was published, I figured I would quickly implement and publish it. Hopefully it is working well for users.

**Comment by rtack** [comment]
wondering how much effort it would be to implement this for slim.
Am I correct that the actual code who does the formatting is at https://github.com/vingorius/pug-beautify/blob/master/index.js ?

**Comment by Glavin001** [comment]
@rtack : Yes the code that formats/beautifies Jade/Pug language is the pug-beautify package you linked to there. You could talk with the maintainer and Slim does look very similar to hopefully you can work out a solution! That would be great!

**Comment by preston** [comment]
+1 for built-in Slim support!

**Comment by fmatosic** [comment]
+1 for built-in Slim support!

**Comment by Glavin001** [comment]
Please note that +1s do not help this feature become reality. Atom-Beautify requires a third-party beautifier to exist and be integrated into Atom-Beautify for use.

With that in mind, how you can turn this feature request into a feature is by:
1) finding a beautifier that works
2) submitting a Pull Request to Atom-Beautify such that you can use it

As noted, `pug-beautify` may work. I recommend that each of you go +1 an issue over at https://github.com/vingorius/pug-beautify/blob/master/index.js (or create an issue if necessary). Once pug-beautify supports Slim, then come back and add support to Atom-Beautify with a Pull Request.
Thank you.
